data/2022_events_db.csv

```diff
@@ -3,30 +3,30 @@ license:	All data in this file is published as Open Data, you are free to reuse
-DevConf.cz	1638149135	yes	20220128	20220129	DevConf.cz		devconf_cz	https://www.devconf.info/cz/	online				On the internet				https://www.devconf.info/coc	<p>DevConf.CZ 2022 is the 14th annual, free, Red Hat sponsored virtual community conference for developers, admins, DevOps engineers, testers, documentation writers and other contributors to open source technologies. The conference includes topics on Linux, Middleware, Virtualization, Storage, Cloud and mobile. At DevConf.CZ, FLOSS communities sync, share, and hack on upstream projects together.</p><p>While we are disappointed that we won’t be able to see you all in person, we're shifting to an awesome virtual event in 2021!</p><p>There is no admission or ticket charge for DevConf.CZ events. However, you are required to complete a free registration. Watch this site for updates about registration.</p><p>We are committed to fostering an open and welcoming environment at our conference. We set expectations for inclusive behavior through our code of conduct and media policies, and are prepared to enforce these.</p>	20211001	https://cfp.devconf.info/	Community Event	Red Hat	Zero	yes	?	EN	Red Hat		2009	Red Hat 		?	?	?	?	https://twitter.com/devconf_cz		https://t.me/devconfcz	https://github.com/devconfcz/devconf		https://www.youtube.com/channel/UCmYAQDZIQGm_kPvemBc_qwg								
+DevConf.cz	1638149135	yes	20220128	20220129	DevConf.cz		devconf_cz	https://www.devconf.info/cz/	online				On the internet				https://www.devconf.info/coc	<p>DevConf.CZ 2022 is the 14th annual, free, Red Hat sponsored virtual community conference for developers, admins, DevOps engineers, testers, documentation writers and other contributors to open source technologies. The conference includes topics on Linux, Middleware, Virtualization, Storage, Cloud and mobile. At DevConf.CZ, FLOSS communities sync, share, and hack on upstream projects together.</p><p>While we are disappointed that we won’t be able to see you all in person, we're shifting to an awesome virtual event in 2021!</p><p>There is no admission or ticket charge for DevConf.CZ events. However, you are required to complete a free registration. Watch this site for updates about registration.</p><p>We are committed to fostering an open and welcoming environment at our conference. We set expectations for inclusive behavior through our code of conduct and media policies, and are prepared to enforce these.</p>	20211001	https://cfp.devconf.info/	Community Event	Red Hat	Zero	yes	?	EN	Red Hat		2009	"Red Hat "		?	?	?	?	https://twitter.com/devconf_cz		https://t.me/devconfcz	https://github.com/devconfcz/devconf		https://www.youtube.com/channel/UCmYAQDZIQGm_kPvemBc_qwg								
```

Comment:
Can you please revert all unrelated changes?

Reply:
Sure, gonna take a look! I think it is because of a Visual Studio Code Addon, which let me edit CSV faster. Maybe I'll need some help, how to update it :D

Reply:
Just push new commits. I will squash them on merge.